src/test/scala/org/apache/mesos/chronos/scheduler/jobs/ScheduleStreamSpec.scala

```diff
@@ -9,61 +9,88 @@ class ScheduleStreamSpec extends SpecificationWithJUnit {
     }
 
     "be convertible back to a string for simple cases" in {
-      val orgSchedule = new ScheduleStream(null, Schedule.parse("R/2012-01-01T00:00:00.000Z/P1D").get)
+      val orgSchedule = new ScheduleStream(null, Schedules.parseIso8601Schedule("R/2012-01-01T00:00:00.000Z/P1D").get)
       val newSchedule = orgSchedule.tail.get.tail.get.head._2
 
-      newSchedule.toZeroOffsetISO8601Representation must_== "R/2012-01-03T00:00:00.000Z/P1D"
+      newSchedule.toStringRepresentation must_== "R/2012-01-03T00:00:00.000Z/P1D"
     }
 
     "handle variable length months repetition correctly" in {
-      val orgSchedule = Schedule.parse("R/2012-01-31T00:00:00.000Z/P1M").get
+      val orgSchedule = Schedules.parseIso8601Schedule("R/2012-01-31T00:00:00.000Z/P1M").get
       val stream = new ScheduleStream(null, orgSchedule)
 
       val nextSchedule = stream.tail.get.head._2
-      nextSchedule.invocationTime must_== Schedule.parse("R/2012-02-29T00:00:00.000Z/P1M").get.invocationTime
-      nextSchedule.offset must_== 1
+      nextSchedule.invocationTime must_== Schedules.parseIso8601Schedule("R/2012-02-29T00:00:00.000Z/P1M").get.invocationTime
+      nextSchedule.asInstanceOf[Iso8601Schedule].offset must_== 1
       nextSchedule.recurrences must_== None
 
       val nextSchedule2 = stream.tail.get.tail.get.head._2
-      nextSchedule2.invocationTime must_== Schedule.parse("R/2012-03-31T00:00:00.000Z/P1M").get.invocationTime
-      nextSchedule2.offset must_== 2
+      nextSchedule2.invocationTime must_== Schedules.parseIso8601Schedule("R/2012-03-31T00:00:00.000Z/P1M").get.invocationTime
+      nextSchedule2.asInstanceOf[Iso8601Schedule].offset must_== 2
 
       val nextSchedule3 = stream.tail.get.tail.get.tail.get.head._2
-      nextSchedule3.invocationTime must_== Schedule.parse("R/2012-04-30T00:00:00.000Z/P1M").get.invocationTime
-      nextSchedule3.offset must_== 3
+      nextSchedule3.invocationTime must_== Schedules.parseIso8601Schedule("R/2012-04-30T00:00:00.000Z/P1M").get.invocationTime
+      nextSchedule3.asInstanceOf[Iso8601Schedule].offset must_== 3
+    }
+
+    "Generate cron schedules the same way" in {
+
+      def utcTime(s: String) = DateTime.parse(s).withZoneRetainFields(DateTimeZone.UTC)
+
+      val now = utcTime("2015-10-19T14:00:00.000Z")
+
+      val oneAM = Schedules.parseCronSchedule("0 1 * * *", currentTime = now).get
+      val stream = new ScheduleStream("name", oneAM)
+
+      stream.head._2.invocationTime must_== utcTime("2015-10-20T01:00:00.000Z")
+      stream.tail.get.head._2.invocationTime must_== utcTime("2015-10-21T01:00:00.000Z")
+      stream.tail.get.tail.get.head._2.invocationTime must_== utcTime("2015-10-22T01:00:00.000Z")
+    }
+
+    "Generate cron schedules for other time zones" in {
+      def chicagoTime(s: String) = DateTime.parse(s).withZoneRetainFields(DateTimeZone.forID("US/Central"))
```

Comment:
The same question as above: would `US/Central` actually be a different timezone in different times of the year, yielding different UTC times?